src/test/java/frc/robot/subsystems/MecanumTest.java

```diff
@@ -1,33 +0,0 @@
-// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
don't delete this file - it's used to demonstrate the unit testing functionality. if the build fails, then this file needs to be modified